modules/virt-creating-vm-instancetype.adoc

```diff
@@ -6,30 +6,29 @@
 [id="virt-creating-vm-instancetype_{context}"]
 = Creating a virtual machine from an instance type
 
-[IMPORTANT]
-====
-This is a Developer Preview feature only. Developer Preview features are not supported by Red Hat in any way and are not functionally complete or production-ready. Do not use Developer Preview features for production or business-critical workloads. Developer Preview features provide early access to upcoming product features in advance of their possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. These features might not have any documentation, are subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview features without an associated SLA.
-====
-
 You can use common instance types to install and edit customized instance types and preferences to create a virtual machine (VM).
 
-.Procedure
+:FeatureName: Creating a virtual machine from an instance type
+include::snippets/technology-preview.adoc[]
 
-. In the web console, navigate to the *InstanceTypes* tab in one of the following ways:
+.Procedure
 
-+
-* Click *Virtualization* -> *Catalog* -> *InstanceTypes*.
-* Click *Virtualization* -> *VirtualMachines* and select *From volume* in the *Create* dropdown menu.
+. In the web console, navigate to *Virtualization* -> *Catalog* -> *InstanceTypes*.
 
 . Select a volume to boot from.
 
-. Select an 'InstanceType'.
+[NOTE]
+====
+The volumes list shows only volumes that exist in the `openshift-virtualization-os-images` namespace and have the `instancetype.kubevirt.io/default-preference` label.
+====
+
+. Select an instance type from the 'InstanceType' list.
```

Comment:
As per Andrea's comments in the PR, are we going with 'InstanceType' or *InstanceType*? If we're choosing for the latter, we probably want to keep it consistent throughout the docs. The following step uses 'InstanceType' and would need fixed. I'm not 100% sure which method we've opted for since that convo.

There is an instance in the following module that bolds InstanceType: 

|Creates a VM using the boot source and settings chosen on the *InstanceTypes* tab


but maybe that's because it's specifically referring to the tab?

Reply:
@stevsmit This is the text updated after the discussion above. "an InstanceType' was updated to "an instance type" for clarity, but the list is still the "InstanceType" list.

Reply:
If "InstanceType" is in the GUI, then it should be bold (which I think is what steven was trying to show above, but a single asterisks renders as italics, not bold in GitHub).